plugin/telescope.vim

```diff
@@ -1,6 +1,7 @@
 
 " Sets the highlight for selected items within the picker.
 highlight default link TelescopeSelection Visual
+highlight default link TelescopeSelectionCaret Visual
```

Comment:
Let's make the default equal to TelescopeSelection, so that it's easier for people to keep their same setup.

Reply:
makes sense, thanks!